### ISSUE TITLE
Bug with $this->classList missing space

### ISSUE THREAD

**Issue by Aybee** [codefog/contao-events_subscriptions#48]
Affects **dev-feature/number-of-participants@dev** - Contao 4.9.4
```html
<div class="event layout_listcurrent subscribed odd last cal_2"></div>
<div class="event layout_list current subscribed odd last cal_2"></div>
```
1. line actual state
2. line should be

I used the original template `event_list_subscribe.html5`

It only appears when you are subscribed (tested only with member subscription). When you are not subcribed it looks ok.
```html
<div class="event layout_list current odd last cal_2">
```
`$this->classList` by Contao always has a leading `SPACE`. In debug mode you can see, that when the member is subscribed, the leading `SPACE` is missing.

**Comment by qzminski** [comment]
This should be fixed in 2.8.4 and on that feature branch :+1:

**Comment by Aybee** [comment]
Thank you very much 👍
